optimum/intel/neural_compressor/quantization.py

```diff
@@ -36,11 +37,11 @@
     WEIGHTS_NAME,
     IncDataLoader,
     _cfgs_to_fx_cfgs,
-    _get_quantizable_ops_recursively,
 )
```

Comment:
`from optimum.intel.neural_compressor.utils import WEIGHTS_NAME, IncDataLoader, _cfgs_to_fx_cfgs` will allow to pass the check_code_quality test

Reply:
done